m01_introduction/01_gallery.rst

```diff
@@ -0,0 +1,158 @@
+.. The roles  should be created at the beginning of the document. Global roles can be created in the `global.rst` file
+
+.. include:: /global.rst
+
+.. Chapter content
+
+Gallery of features
+===================
+
+.. styled-topic::
+
+  Main questions:
+    What are the main features of A+?
+
+  Topics:
+    In this section we will talk about:
+
+    * `Front page`_
+    * `Exercises`_
+    * `Services`_
+
+  Material:
+    In this chapter you won't need extra material.
+
+  Requirements:
+    There are no requirements for this chapter.
+
+  Estimated working time:
+    20min.
+
+
+
+This section is a visual index of exercises and services that have been used
+with A+.
+
+.. _front-page:
+
+Front page
+----------
+
+.. figure:: /images/gallery/aplus-front-page.png
```

Comment:
The image shows hidden courses. It would only show visible courses if you were logged out from A+.